regimes/se/tax_identities.go

```diff
@@ -0,0 +1,79 @@
+package se
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+const (
+	taxCodeCountryPrefix = "SE"
+	taxCodeLength        = 14
+	taxCodeCheckDigit    = "01"
+)
+
+var (
+	// ErrInvalidTaxIDLength is returned when the tax ID is not the correct length.
+	ErrInvalidTaxIDLength = errors.New("invalid length")
+	// ErrInvalidTaxIDCountryPrefix is returned when the tax ID does not start with "SE".
+	ErrInvalidTaxIDCountryPrefix = errors.New("invalid country prefix")
+	// ErrInvalidTaxIDCheckDigit is returned when the tax ID does not end with "01".
+	ErrInvalidTaxIDCheckDigit = errors.New("invalid check digit")
+	// ErrInvalidTaxIDCharacters is returned when the tax ID contains non-numeric characters.
+	ErrInvalidTaxIDCharacters = errors.New("invalid characters, expected numeric")
+)
+
+// normalizeTaxIdentity performs normalization specific to Swedish tax IDs,
+// ensuring the code is normalized and the country prefix is added if missing.
+func normalizeTaxIdentity(id *tax.Identity) {
+	tax.NormalizeIdentity(id)
+	// TO-DO: decide if this is necessary. If not, we may need to remove the check digit suffix.
+	// Re-add the SE prefix if missing.
+	if id.Code.String()[:2] != taxCodeCountryPrefix {
+		id.Code = cbc.Code(taxCodeCountryPrefix + id.Code.String())
+	}
```

Comment:
In GOBL, all tax identities are normalized **without** the leading country code. The `tax.Identity`'s `String()` method will always add the country code up front.

Reply:
My only concern is that in the case of other countries (for example Spain) that prefix is optional (you could just include a business' CIF without `ES` and still be valid). However in Sweden, it **must** always be there, hence why I'm on the fence. Should I still remove it regardless?